staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -422,13 +422,12 @@ func createHandler(r rest.NamedCreater, scope RequestScope, typer runtime.Object
 			}
 		}
 
+		// TODO: replace with content type negotiation?
+		bypassInitialization := req.URL.Query().Get("includeUninitialized") == "1"
```

Comment:
bypassing initialization and just not waiting for initialization seem like very different concepts?